apps/docs/app/features/routes/ressurser/design-tokens/SpacingTokens.tsx

```diff
@@ -13,75 +13,10 @@ import {
   Tr,
 } from "@vygruppen/spor-react";
 import { Fragment } from "react";
-import tokens from "@vygruppen/spor-design-tokens";
 import { useTokenFormatter } from "~/features/tokens/useTokenFormatter";
+import { SharedTokenLayout } from "./SharedTokenLayout";
 
-type SpacingToken = {
-  name: string;
-  key:
-    | "3"
-    | "6"
-    | "9"
-    | "12"
-    | "18"
-    | "24"
-    | "30"
-    | "36"
-    | "42"
-    | "54"
-    | "72"
-    | "90";
-};
-const spacingTokens: SpacingToken[] = [
-  {
-    name: "0.5",
-    key: "3",
-  },
-  {
-    name: "1",
-    key: "6",
-  },
-  {
-    name: "1.5",
-    key: "9",
-  },
-  {
-    name: "2",
-    key: "12",
-  },
-  {
-    name: "3",
-    key: "18",
-  },
-  {
-    name: "4",
-    key: "24",
-  },
-  {
-    name: "5",
-    key: "30",
-  },
-  {
-    name: "6",
-    key: "36",
-  },
-  {
-    name: "7",
-    key: "42",
-  },
-  {
-    name: "8",
-    key: "54",
-  },
-  {
-    name: "9",
-    key: "72",
-  },
-  {
-    name: "10",
-    key: "90",
-  },
-];
+const { px: spacingPxs, ...spacingSizes } = tokens.size.spacing;
```

Comment:
Here I'm separating the px spacing tokens from the rest of the spacing tokens (which are sm, md, lg etc)